packages/fiori/src/Timeline.js

```diff
@@ -96,6 +120,19 @@ class Timeline extends UI5Element {
 
 		this._itemNavigation.setCurrentItem(target);
 	}
+
+	onBeforeRendering() {
+		this._itemNavigation.navigationMode = this.layout === TimelineLayout.Horizontal ? NavigationMode.Horizontal : NavigationMode.Vertical;
+
+		for (let i = 0; i < this.items.length; i++) {
+			this.items[i].layout = this.layout;
+			if (this.items[i + 1] && !!this.items[i + 1].icon) {
+				this.items[i]._lineWidth = "short";
```

Comment:
this property can be converted to boolean and can be renamed to shortLineWidth (for example) in order not to make hard-coded string comparison in ```get classes```

Reply:
Done